tools/cldr-code/src/main/java/org/unicode/cldr/util/VettingViewer.java

```diff
@@ -1265,7 +1265,7 @@ public static MissingStatus getMissingStatus(CLDRFile sourceFile, String path, b
 
         String value = sourceFile.getStringValue(path);
         Status status = new Status();
-        sourceFile.getSourceLocaleID(path, status);
+        String sourceLocale = sourceFile.getSourceLocaleID(path, status);
         boolean isAliased = !path.equals(status.pathWhereFound); // this was path.equals, which would be incorrect!
```

Comment:
sourceLocale looks unused

Reply:
It is unused . I just assigned it because that way it was easier to debug I think the compiler will just ignore it

Reply:
The compiler will issue a warning, depending on config -- one option would be if (DEBUG) ... println ... sourceLocale

Very minor quibble! I thought I'd better check anyway that the assignment wasn't for the sake of something intended but missing.

Reply:
quibble to the quibble: rather than if(DEBUG), use `logger.finest(() -> "sourceLocale:"+sourceLocale);`

Reply:
I looked at https://www.geeksforgeeks.org/logger-finest-method-in-java-with-examples/, and that is a lot of function calls for something that is only needed when debugging. So I suggest we not entangle this PR, and that you put together a short doc about how and where we could deploy logging, and what the performance implications are, and we can discuss in the cldr infra meeting.

Reply:
Steven's the expert on the new logging features but I think it would just entail adding one line in VettingViewer:
```
    private static final Logger logger = SurveyLog.forClass(VettingViewer.class);
```
Anyway that's low priority.

Reply:
yes, that will be documented in CLDR-5313